tensorflow/python/feature_column/feature_column_v2.py

```diff
@@ -1026,19 +1026,47 @@ def numeric_column(key,
 
   Example:
 
-  ```python
-  price = numeric_column('price')
-  columns = [price, ...]
-  features = tf.io.parse_example(..., features=make_parse_example_spec(columns))
-  dense_tensor = input_layer(features, columns)
-
-  # or
-  bucketized_price = bucketized_column(price, boundaries=[...])
-  columns = [bucketized_price, ...]
-  features = tf.io.parse_example(..., features=make_parse_example_spec(columns))
-  linear_prediction = linear_model(features, columns)
-  ```
-
+  Assume we have a dataset with two features `a` and `b`.
+  Feature `a` is of numeric in nature.
+  
+  >>> data = {'a': [15, 9, 17, 19, 21, 18, 25, 30],
+  ...    'b': [5.0, 6.4, 10.5, 13.6, 15.7, 19.9, 20.3 , 0.0]}
+  
+  Convert the data to `tf.data.Dataset` and batch the dataset.
```

Comment:
Using `tf.data` doesn't add anything here.
How about just:

```
feature_layer(data)
```

Reply:
Okay, I will remove it. My thought process of using `tf.data`  was to show a close to real world scenario where the end user would have a `tf.data.Dataset` in place of a dictionary.

Reply:
Implemented.